src/category_theory/limits/shapes/terminal.lean

```diff
@@ -33,7 +33,17 @@ class has_initial :=
 
 attribute [instance] has_terminal.has_limits_of_shape has_initial.has_colimits_of_shape
 
+/--
+The chosen terminal object, if it exists.
+You can use the notation `⊤_ C`.
+This object is characterized by having a unique morphism from any object.
+-/
 abbreviation terminal [has_terminal C] : C := limit (functor.empty C)
+/--
+The chosen initial object, if it exists.
+You can use the notation `⊥_ C`.
+This object is characterized by having a unique morphism from any object.
```

Comment:
I think there is a copy-paste typo here:
```suggestion
This object is characterized by having a unique morphism to any object.
```

Reply:
Thanks! Fixed.